packages/js/components/src/phone-number-input/index.tsx

```diff
@@ -0,0 +1,219 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useRef, useLayoutEffect } from 'react';
+import { createElement } from '@wordpress/element';
+import { useSelect } from 'downshift';
```

Comment:
Is downshift being included in the WP environment via external dependency or is it bundled directly into the built JS?

The former means we have roughly 52kb (non-gzipped) size impact on page load.

Reply:
AFAIK it's not part of the [WP](https://www.npmjs.com/package/@wordpress/dependency-extraction-webpack-plugin)/[WC](https://www.npmjs.com/package/@woocommerce/dependency-extraction-webpack-plugin) extraction list. So I imagine it should be directly bundled, although I don't see it using webpack bundle analyzer 🤔 But it's hard to dig deeper with the webpack JSON profile being so big 😅 

On the other hand, it's also used by [`select-control`](https://github.com/woocommerce/woocommerce/blob/93b6f1b8ce5b27064ef9745cf4f49026365b114e/packages/js/components/src/experimental-select-control/select-control.tsx) here, and `@wordpress/components` [`custom-select-control`](https://github.com/WordPress/gutenberg/blob/58f5e9a6a972d16aeca13693ce863bcedaf342b0/packages/components/src/custom-select-control/index.js)